### PR TITLE
Add GlobalAlertTemplates for ADJobs

### PR DESCRIPTION
**Pull request by freecaykes**
## Description
Creates GlobalAlert Templates for AD Jobs as part of the render/intrusion_detection pkg  

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [x] Milestone set according to targeted release.
- [x] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -41,6 +40,10 @@ replace (
 	github.com/go-logr/zapr => github.com/go-logr/zapr v0.2.0
 	//github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.1
 	github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v1.0.1
+
+	// FIXME: for official version
```
Comment (freecaykes) [inline go.mod:44]: - [ ] remove once official tigera/api for new GlobalAlert fields is released

**Review by doublek — COMMENTED** [review]
Couple of reminders to cleanup. The rest looks good. Should get @Brian-McM to approve from operator maintainers perspective as well since he was involved (a while back) in the design discussion.

File: go.mod [hunk 2]
```diff
@@ -38,6 +38,7 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
 	github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v1.0.1
 
+	github.com/tigera/api => github.com/freecaykes/api v0.0.0-20220226014818-38cb42b095ad
```
[the remark this answers is not in the document]
Comment (freecaykes) [reply go.mod]: done with rebase

File: pkg/crds/calico/crd.projectcalico.org_bgpconfigurations.yaml [hunk 3]
```diff
@@ -66,6 +66,37 @@ spec:
                 description: 'LogSeverityScreen is the log severity above which logs
                   are sent to the stdout. [Default: INFO]'
                 type: string
+              nodeMeshMaxRestartTime:
+                description: Time to allow for software restart for node-to-mesh peerings.  When
+                  specified, this is configured as the graceful restart timeout.  When
+                  not specified, the BIRD default of 120s is used. This field can
```
[the remark this answers is not in the document]
Comment (freecaykes) [reply pkg/crds/calico/crd.projectcalico.org_bgpconfigurations.yaml]: done with rebase

[the remark this answers is not in the document]
Comment (freecaykes) [reply pkg/crds/calico/crd.projectcalico.org_bgpconfigurations.yaml]: cleaned up with another `make gen-versions` https://github.com/tigera/operator/pull/1600/commits/aa896ecc85287787c86317d81752c7fb33f7214f

**Review by doublek — APPROVED** [review]
LGTM. Should get a sign off from @Brian-McM as well.

**Review by Brian-McM — APPROVED** [review]

**Pull request merged**